.github/workflows/ci.yml

```diff
@@ -3,20 +3,25 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  build:
+  test:
+    name: test w/ Python
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.10"]
```

Comment:
Did you check that tests are being run with these Python versions?

Reply:
Yes, there is just one warning:
<img width="1026" alt="image" src="https://user-images.githubusercontent.com/13194519/154724838-911f7752-2079-48a0-9225-c7b6fdb9505e.png">

<img width="1022" alt="image" src="https://user-images.githubusercontent.com/13194519/154725163-1367ff07-4e16-4210-b651-2b09527b021a.png">

Reply:
Thank you for pointing out that problem. Now the CI is working with Python 3.7 and 3.10. There is no warning reported on Python 3.7, and [two warnings](https://github.com/airspeed-velocity/asv/runs/5259920032?check_suite_focus=true) on Python 3.10. I'm going to check it in another PR.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/13194519/154812351-73fb41b6-0f38-480e-8dc3-8c9d9f476f52.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/13194519/154812456-1af8f179-8606-4d98-84c3-c1744d2f2772.png">